dll/shellext/fontext/CFontExt.cpp

```diff
@@ -601,8 +597,31 @@ HRESULT CFontExt::DoInstallFontFile(LPCWSTR pszFontPath, LPCWSTR pszFontsDir, HK
     return S_OK;
 }
 
-HRESULT CFontExt::DoGetFontTitle(LPCWSTR pszFontPath, LPCWSTR pszFontName)
+typedef BOOL (WINAPI *FN_GetFontResourceInfoW)(LPCWSTR, DWORD *, void*, DWORD);
+
+HRESULT
+CFontExt::DoGetFontTitle(IN LPCWSTR pszFontPath, OUT LPWSTR pszFontName,
+                         IN DWORD cbFontName)
 {
-    // TODO:
+    HMODULE hGdi32 = LoadLibraryA("gdi32");
```

Comment:
Why not define the prototype for the function and directly call it, why LoadLibrary/GetProcAddress?

Reply:
OK.